src/peer_channels.rs

```diff
@@ -480,13 +486,16 @@ pub async fn send_webxdc_realtime_data(ctx: &Context, msg_id: MsgId, data: Vec<u
 
 /// Leave the gossip of the webxdc with given [MsgId].
 pub async fn leave_webxdc_realtime(ctx: &Context, msg_id: MsgId) -> Result<()> {
-    if !ctx.get_config_bool(Config::WebxdcRealtimeEnabled).await? {
+    let Some(iroh) = ctx.get_peer_channel().await else {
+        warn!(
+            ctx,
+            "IROH_REALTIME: tried to leave channel, but Iroh is already not initialized {msg_id}"
+        );
         return Ok(());
-    }
+    };
     let topic = get_iroh_topic_for_msg(ctx, msg_id)
         .await?
         .with_context(|| format!("Message {msg_id} has no gossip topic"))?;
```

Comment:
This shouldn't be an error then, otherwise the app have to handle this unnecessarily. If there's no gossip topic, we can just return `Ok(())`

Reply:
Makes sense. I'll turn this into OK then.

Reply:
OK, looks like you did it.